content/chainguard/chainguard-images/images-testing.md

```diff
@@ -0,0 +1,72 @@
+---
+title: "How Chainguard Images are Tested"
+linktitle: "How Images are Tested"
+type: "article"
+description: "A conceptual article outlining testing requirements for Chainguard Images."
+date: 2024-03-13T11:07:52+02:00
+lastmod: 2023-03-13T11:07:52+02:00
+draft: false
+tags: ["Overview", "Chainguard Images", "Product"]
+images: []
+menu:
+  docs:
+    parent: "chainguard-images"
+weight: 550
+toc: true
+---
+
+Chainguard Images are minimal, distroless container images that you can use to build and run secure applications. A frequently-asked question about these Images relates to what kind of testing Chainguard performs on them to ensure that they are indeed smaller and more secure than external counterparts.
+
+This article provides a high level overview of what testing Chainguard performs when building new Images to ensure their security and that they provide an experience consistent with their external counterparts.
+
+
+## Build requirements for new Images
+
+Chainguard has a set of requirements in place that new Images must meet in order to be included in our library of Images. These requirements fall into two categories:
+
+* Image standards
+* Application testing
+
+
+### Image standards
+
+When building a new Image, Chainguard will take steps to ensure it meets the following standards:
+
+| **Requirement** 	  |  **Explanation**     |
+| --- | --- |
+| **Size**     |  Any new Chainguard Images should be smaller than their external counterparts. Any Images that aren't smaller than their counterparts must include an explanation as to why.     |
+|  **CVEs**     | When scanned with a CVE scanning tool like Grype, new Images should return zero CVEs. Again, if an Image does return CVEs in its scan results it should include an explanation. 	  |
+|  **Kubernetes accessibility**     | Containers based on Images must be able to run inside of a Kubernetes cluster.     |
+|  **Architecture**     | Chainguard Images must be built for both the `x86_64` and `aarch64` architectures.      |
+|  **Tags**     | Chainguard Image tags must be configured correctly and align with their respective versions.      |
+
+
+### Application testing
+
+Chainguard performs the following checks on new Images to ensure that the applications contained within them meet the needs of most use cases:
+
+| **Requirement** 	  |  **Explanation**     |
+| --- | --- |
+| **Application startup**     |  When a container based on a Chainguard Image is launched, the Image's application starts correctly.    |
```

Comment:
I would combine `Application startup`, `Functionality`, and `Helm chart compatbility` into just `Functionality`.

Since functionality varies based on the image, I think it makes more sense to bin it there, maybe something like:

```
|  **Functionality**     | The application is tested to comply with it's upstream counter parts core feature set.   |
```